spec/asyncapi.md

```diff
@@ -2554,6 +2584,6 @@ string | `string` | | |
 byte | `string` | `byte` | base64 encoded characters
 binary | `string` | `binary` | any sequence of octets
 boolean | `boolean` | | |
-date | `string` | `date` | As defined by `full-date` - [RFC3339](https://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14)
-dateTime | `string` | `date-time` | As defined by `date-time` - [RFC3339](https://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14)
+date | `string` | `date` | As defined by `full-date` - [RFC3339](https://www.rfc-editor.org/rfc/rfc3339)
+dateTime | `string` | `date-time` | As defined by `date-time` - [RFC3339](https://www.rfc-editor.org/rfc/rfc3339)
```

Comment:
@derberg I fixed these two URLs and the Markdown links Action is still complaining. Any idea?

Reply:
LOL, funny. I tried it in [this commit](https://github.com/asyncapi/spec/pull/827/commits/82a335fd61fa376980c2460c5f5cb54021bca183) and didn't work. But the funniest thing is that there's a 404 on https://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14 but that's exactly the URL I replaced in this PR. It shouldn't be checking for its existence, right?

Reply:
It seems the same issue is happening here as well: https://github.com/asyncapi/parser-js/pull/601#issuecomment-1240468666

Reply:
yeah, looks like there are some issues with action that we are using,
I'll open a PR to `.github` repo to use their new pre release

in the meantime, remove your changes of links as they will be fixed with https://github.com/asyncapi/spec/pull/832